source/ap.js

```diff
@@ -38,7 +38,11 @@ var ap = _curry2(function ap(applyF, applyX) {
         ? applyF.ap(applyX)
         : typeof applyF === 'function'
           ? function(x) { return applyF(x)(applyX(x)); }
-          : _reduce(function(acc, f) { return _concat(acc, map(f, applyX)); }, [], applyF)
+          : _reduce(
+            function(acc, f) { return _concat(acc, map(f, applyX)); },
+            [],
+            applyF
+          )
```

Comment:
Post `1.0`, I would like to do some auto-formatting, using Prettier or something, and from then on simply ignore such things.  The trouble, of course, is that auto-formatters are mostly designed around OOP/imperative styles, and those don't always transfer well to FP/declarative code.  Still, I would love to simply never have to make such judgement calls.

Reply:
My bad. I'm gonna revert this formatting part.